src/lib/containers/entity/annotations/SchemaDrivenAnnotationEditor.tsx

```diff
@@ -39,18 +39,16 @@ export type SchemaDrivenAnnotationEditorProps = {
   entityId?: string
   /** If no entity ID is supplied, the schema to use for the form */
   schemaId?: string
+  /** Optionally supply a ref to the form to handle submission externally with `formRef.current.submit()` */
+  formRef?: React.RefObject<Form<Record<string, unknown>>>
```

Comment:
Allows us to invoke submit outside of the editor, e.g. from the "Save" button in a modal footer

Reply:
Perhaps expand this comment even further, indicating that if supplied then caller must handle submission/cancellation (and provide the Save/Cancel UI).